share/revng/test/configuration/revng-qa/for-runtime-abi-tests.yml

```diff
@@ -209,15 +259,59 @@ commands:
   - type: revng-qa.runtime-abi-tests
     from:
       - type: source
-        filter: for-runtime-abi-tests and !windows
+        filter: for-runtime-abi-tests and !windows and !clang-only
     suffix: /
     command: |-
-      "${INPUT}/build_abi_test_binaries_with_gcc.sh"
+      source "${INPUT}/prepare-functions-binary.sh" &&
```

Comment:
No need for `&&`, we should be `set -e`.

Reply:
Done.